README.md

```diff
@@ -1,113 +1,32 @@
-<p align="center">
-  <a href="https://www.gatsbyjs.com">
-    <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
-  </a>
-</p>
-<h1 align="center">
-  Starter for a Gatsby Plugin
-</h1>
+# gatsby-source-apostrophe
 
-A minimal boilerplate for the essential files Gatsby looks for in a plugin.
+Source plugin for accessing ApostropheCMS content APIs in Gatsby
```

Comment:
Should specify off the bat it's for A3, yes?

Reply:
True.